### ISSUE TITLE
tests, test?

### ISSUE THREAD

**Issue by tomchen** [pypa/sampleproject#133]
pypa/sampleproject and [Python Packaging User Guide's "Packaging Python Projects" page](https://packaging.python.org/tutorials/packaging-projects/?highlight=test#creating-the-package-files) suggest the unit test folder be named `tests`. But by default, [all files matching the pattern **test**/test*.py](https://packaging.python.org/guides/using-manifest-in/?highlight=test#how-files-are-included-in-an-sdist), not **tests**/test*.py, are included in the source distribution. Indeed, I renamed `tests` to `test` in sampleproject, then `python setup.py sdist bdist_wheel` will create a `dist` package that includes test/test*.py.

Or, did sampleproject intentionally change the name to avoid sdist's inclusion of the test files? That'd be a weird way to do so. It would be better to rename it back to `test` and explicitly put an `exclude` line in `MANIFEST.in` if you want the exclusion.

**Comment by dmtucker** [comment]
`tests` is far more common in my experience. I'm kinda surprised to hear `test` is included in sdists by default... not sure why that would be desirable.

**Comment by chrahunt** [comment]
The only reason I know of for naming the folder `tests` is to avoid conflicting with the [`test`](https://docs.python.org/3/library/test.html) standard library module.

**Comment by tomchen** [comment]
Yeah right. It should be a [pypa/setuptools](https://github.com/pypa/setuptools) issue, I opened a PR there.

Meanwhile, there is still something sampleproject could do: maybe add a line `include tests/test*` to `MANIFEST.in` to ask setuptools for the inclusion of `tests`?

**Comment by tomchen** [comment]
The PR there (pypa/setuptools#2494) got closed for some... technical reasons, it is complicated to add `tests/test*` until other problems are solved...

So I guess here's what we can do now:
* In sampleproject, add a line `include tests/test*` to `MANIFEST.in`
* Add some notice to ["Including files in source distributions with MANIFEST.in" page in Python Packaging User Guide](https://packaging.python.org/tutorials/packaging-projects/?highlight=test#creating-the-package-files), explaining why `test` there contradicts `tests` in the same user guide's ["Packaging Python Projects" page](https://packaging.python.org/tutorials/packaging-projects/) and in pypa/sampleproject, and suggest what users should do, what the folder name and what sdist inclusion strategy might be good.

**Comment by di** [comment]
The directory in this project should probably just change to `test` so that it aligns with the default `setuptools` behavior.
